index.html

```diff
@@ -0,0 +1,82 @@
+      <label id="number-label">Age</label>
+      <input id="number"  type="number" placeholder="Age" min="15" max="100"> <br>
+      <label id="dropdown">Which option best describes your current role?</label> <br>
+      <select id="dropdown">
+        <option value="" > Select current role</option>
+        <option value="student">Student</option>
+        <option value="full-time-learner">Full time learner</option>
+        <option value="full-time-job">Full Time Job</option>
+        <option value="prefer-not-to-say">Prefer not to say</option>
+      </select>
+      <br>
+      <label id="radio">Would you recommend freeCodeCamp to a friend?</label>
+      <input type="radio" name="option" id="definitely" value="definitely" checked>
+      <label for="definitely">Definitely</label> <br>
+      <input type="radio" name="option" id="maybe" value="maybe">
+      <label for="maybe">Maybe</label> <br>
+      <input type="radio" name="option" id="notsure" value="notsure">
+      <label for="notsure">Not sure</label> <br>
+
+      <label id="dropdown">What is your favorite feature of freeCodeCamp?</label> <br>
+      <select id="dropdown-label">
+        <option>Select an option</option>
+        <option>Challenges</option>
+        <option>Projects</option>
+        <option>Community</option>
+        <option>Open Source</option>
+      </select>
+      <br>
+      <label for="check-apply">What would you like to see improved? (Check all that apply)</label> <br>
+      <label for="front-end">  <input type="checkbox" id="front-end" value="front-end" name="front-end"> Front-end Projects</label> <br>
+      <label for="back-end"> <input type="checkbox" id="back-end" value="back-end" name="back-end"> Back-end Projects</label> <br>
+      <label for="data-visual"><input type="checkbox" id="data-visual" value="data-visual" name="data-visual">  Data Visualization</label> <br>
+      <label for="challenges"><input type="checkbox" id="challenges" value="challenges" name="challenges">  Challenges</label> <br>
+      <label for="open-source-coummnity"> <input type="checkbox" id="open-source-coummnity" value="open-source-coummnity" name="open-source-coummnity">  Open Source Coummnity </label> <br>
+      <label for="gitter"> <input type="checkbox" id="gitter" value="gitter" name="gitter">  Gitter help rooms</label> <br>
+      <label for="videos"> <input type="checkbox" id="videos" value="videos" name="videos"> Videos</label> <br>
+      <label for="city-meetups"> <input type="checkbox" id="city-meetups" value="city-meetups" name="city-meetups"> City Meetups</label> <br>
+      <label for="wiki"> <input type="checkbox" id="wiki" value="wiki" name="wiki">  Wiki</label> <br>
+      <label for="forum"> <input type="checkbox" id="forum" value="forum" name="forum"> Forum </label> <br>
+      <label for="addtional-courses"> <input type="checkbox" id="addtional-courses" value="addtional-courses" name="addtional-courses"> Additional Courses</label>
+
```

Comment:
- [ ] Kindly remove unnecessary blank lines

Reply:
- [ ] Great job but I think that you forget to give a reason why you are asking this request because maybe it is obvious to you but not for the others 💯🚀🚀

Reply:
- [ ] Hi @Adetunbosun  As the previous reviewer mentioned, I would like to recommend you always tell the reason, **WHY** you suggest something or request a change. Give a good explanation in order to make him/her understand and also showcase your technical knowledge  :smile: